routes/console.php

```diff
@@ -14,10 +14,84 @@
 |
 */
 
-Artisan::command('inspire', function () {
-    $this->comment(Inspiring::quote());
+Artisan::command('allegro-token', function () {
```

Comment:
To jest całkiem spore i może się przydać póxniej proponuję dodać to jako normalną komendę